tools/run-tests.py

```diff
@@ -68,10 +68,7 @@ def skip_if(condition, desc):
             OPTIONS_COMMON + OPTIONS_DOCTESTS + OPTIONS_PROFILE_ES51 + OPTIONS_DEBUG),
     Options('unittests-es5.1-debug-init-fini',
             OPTIONS_COMMON + OPTIONS_UNITTESTS + OPTIONS_PROFILE_ES51 + OPTIONS_DEBUG
-            + ['--cmake-param=-DFEATURE_INIT_FINI=ON'],
-            skip=skip_if((sys.platform == 'win32'), 'FEATURE_INIT_FINI build flag isn\'t supported on Windows,' +
-                         ' because Microsoft Visual C/C++ Compiler doesn\'t support' +
-                         ' library constructors and destructors.')),
+            + ['--cmake-param=-DFEATURE_INIT_FINI=ON']),
```

Comment:
Can you clarify somehow that the build and the related unit test is passing?

Reply:
It's currently just the build flags are supported, didn't related to the unit testing yet.